docs/shared/ungrib.yaml

```diff
@@ -0,0 +1,13 @@
+ungrib:
+  execution:
+    mpicmd: srun
+    batchargs:
+      nodes: 1
+      walltime: 00:05:00
+    executable: "path/to/ungrib.exe"
+  gfs_file: "/path/to/dir/gfs.2024.03.18.12"
+  run_dir: "/path/to/dir/run"
+  vtable: "/path/to/Vtable.GFS"
+platform:
+  account: wrfruc
+  scheduler: slurm
```

Comment:
We probably don't want the `platform` block in this file, as it has its own documentation.